src/main/java/http/utils/MainArgumentParser.java

```diff
@@ -0,0 +1,35 @@
+package http.utils;
+
+public class MainArgumentParser {
+    private int portNumber;
+    private String workingDirectory;
+
+    public MainArgumentParser(String[] args) {
+        portNumber = 5000;
+        workingDirectory = "./";
```

Comment:
I don't like that if I want to change the default port or file directory I have to go-to a class in the utilities folder. This is a reason to change that doesn't belong here.